### ISSUE TITLE
Accessibility Bug: When the ‘close’ button inside the dialog is accessed, reading is made as ‘banner region button’. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3118]
Fundamental core component: "Dailog"

Description: When the ‘close’ button inside the dialog is accessed, reading is made as ‘banner region button’. 

Expected: Additional information ‘banner region’ should be removed when the ‘Close’ button is accessed.

Attached the screenshot
![image](https://user-images.githubusercontent.com/32538291/91555250-430f0480-e94e-11ea-9b98-66da8fd2eb30.png)

Note: - Issue persists for all the ‘Close’ button present within the ‘Dialog’ control.
